### DISCUSSION TITLE
How to work with multiple react quill editors on the same page

### DISCUSSION THREAD

**Discussion by PhucT1006** [slab/quill#4154] [Q&A]
Hi, I'm quite new to using React, and React Quill. I'm trying to get the user to type into 4 different quill editors and then use getText() and getContent() on the 4 editors. However, I don't know how to specify which editors to pull from. I've given each editor an id (editor1, editor2, editor3, editor4). Then when I try to pull the content from each editor I will get "Cannot read properties of undefined" 
Thank you in advance. I appreciate any help.  

![image](https://github.com/quilljs/quill/assets/97328577/273bf8cb-c19d-4c6f-88ce-0cc65edd28ba)

![image](https://github.com/quilljs/quill/assets/97328577/156fe402-bb25-4009-8fb1-fa5cb3019096)

**Comment by shareefalis** [comment]
You are not using refs in your react quill instance.

**Reply by PhucT1006** [reply]
I tried adding that but now the program is saying that getText() and getContent() are not functions
